src/e2e/sdk_remote_integration.spec.ts

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: Apache-2.0
+"use strict";
+
+import { TxReceipt } from "#erdstall/api/responses";
+import { Mint } from "#erdstall/api/transactions";
+import { Address } from "#erdstall/ledger";
+import { Assets } from "#erdstall/ledger/assets";
+import * as assets from "#erdstall/ledger/assets";
+import * as sdk from "#erdstall";
+import { Erdstall } from "#erdstall";
+
+import { ethers } from "ethers";
+import * as fs from "fs";
+import { exec } from "child_process";
+
+(global as any).WebSocket = require("ws");
+
+describe("Erdstall-TS-SDK", () => {
+	const opPort = 1433;
+	const opAddr = new URL(`ws://127.0.0.1:${opPort}/ws`);
+	const nodePort = 1362;
+	const mnemonic = "pistol kiwi shrug future ozone ostrich match remove crucial oblige cream critic";
+
+	before(async () => {
+		await new Promise(accept => fs.writeFile("operator.cfg", JSON.stringify({
+			Mnemonic: mnemonic,
+			OperatorDerivationPath: "m/44'/60'/0'/0/0",
+			EnclaveDerivationPath:  "m/44'/60'/0'/0/1",
+			EpochDuration: 2,
+			ResponseDuration: 3,
+			PowDepth: 0,
+			RPCPort: opPort,
+			RPCHost: "0.0.0.0",
+			NodeReqTimeout: 10,
+			WaitMinedTimeout: 60
+		}), accept));
+
+		exec(`$ERDSTALL_EXE -config operator.cfg -epochs 10 -ledger-port ${nodePort} &`);
```

Comment:
Shouldn't you also cleanup the operator in an `after` statement? How do you make sure that this operator is stopped after the test ends or fails?

Reply:
The operator terminates after 10 epochs.

Reply:
And I have no idea how to terminate the process manually.